### PR TITLE
Fix zoom handling + add readme

### PR DESCRIPTION
**Pull request by jgke**

### CODE REVIEW

**Comment by Violet-CLM** [comment]
It's only going to be bin/Debug if that's how you're building. Otherwise it'll be bin/Release.

**Comment by jgke** [comment]
Yeah, I was thinking on whether I should add a note about that -- that was the minimal amount of steps (?) to get it running so I left it at that. I guess I'll add a note about that?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Building
+--------
+
+1) Open Renderer1.csproj using Visual Studio
+2) Add references to all dll's except bass.dll under the 'Other files to distribute' folder
+   * Right click 'Renderer1' in the Solution Explorer view -> Add -> Reference -> browse
+3) Build (Right click Renderer1 in solution explorer -> Build solution)
+4) Copy all files from 'Other files to distribute' to bin/Debug
+5) Copy MLLE.ini from your JJ2 installation here
+6) Run
```
Comment (jgke) [inline README.md:47]: so something around
```
7) Building release:
   * Change the build from 'Debug' to 'Release'
   * Build solution
   * Copy files from 'Other files to distribute' to bin/Release
```

**Comment by Violet-CLM** [comment]
The installation process could probably be improved in some way, like putting in more useful paths for the libraries in the first place, but I think the important thing here is getting the scrollbar change in, so whatever.

**Pull request merged**